src/app/wallets/update-pending-invoices.ts

```diff
@@ -113,10 +121,22 @@ const updatePendingInvoiceBeforeFinally = async ({
 
   const {
     lnInvoice: { description },
-    roundedDownReceived,
+    roundedDownReceived: uncheckedRoundedDownReceived,
   } = lnInvoiceLookup
 
   // TODO: validate roundedDownReceived as user input
+  const roundedDownReceived = checkedToSats(uncheckedRoundedDownReceived)
```

Comment:
Is this the correct boundary for this validation?

I opted to do it here instead of inside `lndService.lookupInvoice` because:
- it feels like `roundedDownReceived == 0` is invalid in this context, but could be valid in other contexts
- if we can successfully fetch an invoiceLookup then we can call `declineHeldInvoice` on it instead of simply "not canceling" the invoice and just deleting the db record for it ([here](https://github.com/GaloyMoney/galoy/pull/1600/commits/f253ac1d6204c50ae66d84fce67438a6bb647123#diff-f02445547836eff187be3a4b54e41d92b4223dd7c304847b688ef3016eec317fR113))